client/sql/dataframewriter.go

```diff
@@ -0,0 +1,79 @@
+package sql
+
+import (
+	"fmt"
+	proto "github.com/apache/spark-connect-go/v_3_4/internal/generated"
+	"strings"
+)
+
+type DataFrameWriter interface {
```

Comment:
Doc :) Same comment on the missing methods for incremental changes.

Reply:
Yes!